### ISSUE TITLE
imod.write does not accept single layer model input

### ISSUE THREAD

**Issue by Manangka** [Deltares/imod-python#27]
In GitLab by @mulderniels on Jan 10, 2019, 16:36

This won't work:

```python
top = dem.assign_coords(layer=1)
bot = top - 300.0
bnd = xr.full_like(top, 1.0).where(is_bnd).fillna(0.0)


model = OrderedDict()
model["bnd"] = bnd
model["top"] = fill_da(top.where(is_land)).where(is_bnd)
model["bot"] = fill_da(bot.where(is_land)).where(is_bnd)
```

With callback:

```python
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-85-5236f17028ae> in <module>
     28 
     29 name = 'niels'
---> 30 imod.write(path=name, model=model, name=name)

D:\projecten\indus\imod-python-master\imod\__init__.py in write(path, model, name, runfile_parameters)
     73 
     74     if runfile_parameters is None:
---> 75         runfile_parameters = run.get_runfile(model, path)
     76 
     77     if name is None:

D:\projecten\indus\imod-python-master\imod\run.py in get_runfile(model, directory)
    786     runfile_parameters["packages"] = packages
    787     runfile_parameters["stress_periods"] = stress_periods
--> 788     runfile_parameters["output"]["shd"] = list(model["bnd"].layer.values)
    789 
    790     if times:

TypeError: iteration over a 0-d array
```

Solve by calling `np.atleast_1d`.

**Comment by Manangka** [comment]
In GitLab by @visr on Jun 4, 2019, 15:13

Is this still an issue? Would you perhaps like to prepare a merge request?

**Comment by Manangka** [comment]
In GitLab by @visr on Sep 6, 2019, 12:56

closed via commit d3c67f478613dd09c5afc8b99dc1e749da2f46a6
